src/views/Result.vue

```diff
@@ -38,7 +33,7 @@ export default {
 	},
 	mounted() {
 		// initial spinner
-		this.$store.dispatch('setLoading', true);
+		this.$store.dispatch('setLoading', {"active": true,"info": true,"infoMessage": "Vaše texty sme zobrali na spracovanie"});
```

Comment:
No need to be quoted, is it? Just `{active:true, info:true}`. And also, please dont forget to use prettier all the time.